rpc-broker/src/msg.c

```diff
@@ -0,0 +1,286 @@
+/*
+ Copyright (c) 2018 AIS, Inc.
```

Comment:
Last time I asked, the AIS copyright line had to be "Copyright (c) <year> Assured Information Security, Inc."
It's also good practice to have a * at the beginning of each line.